quark/Evaluator/pyeval.py

```diff
@@ -55,8 +52,7 @@ def __invoke(self, instruction):
                     executed_fuc + "(" + ",".join(value_of_reg_list) + ")"
                 )
         # push the return value into ret_stack
-        self.ret_stack.append(
-            executed_fuc + "(" + ",".join(value_of_reg_list) + ")")
+        self.ret_stack.append(executed_fuc + "(" + ",".join(value_of_reg_list) + ")")
```

Comment:
How about this:
`self.ret_stack.append(f"{executed_fuc}({",".join(value_of_reg_list)})}")`

Reply:
LGTM